lession_10/test_homework_10.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+from unittest.mock import patch
+import sys
+import os
+sys.path.append('/home/bardin/PycharmProjects/My project/AQA-Python-Hillel/lession_10')
+
+from homework_10 import log_event
+
+
+class TestLogEvent(unittest.TestCase):
```

Comment:
I ask you use pytest framework instead of unittest. Pytest is one of best practices for Test Automation Engineers as main test automation framework.